From: Stephen Hemminger <stephen.hemminger@vyatta.com>
Date: Tue, 24 Mar 2009 22:40:54 +0000 (-0700)
Subject: Fix headers needed for gre
X-Git-Tag: v2.6.29-1^0
X-Git-Url: http://git.kernel.org/?p=linux%2Fkernel%2Fgit%2Fshemminger%2Fiproute2.git;a=commitdiff_plain;h=1957a322c9932e1a1d2ca1fd37ce4b335ceb7113;hp=a136af4ba6784c3310dee58ee006263336a8b462

Fix headers needed for gre

Need to get right headers or build breaks.
---

diff --git a/ip/link_gre.c b/ip/link_gre.c
index 9109312..9f8bde6 100644
--- a/ip/link_gre.c
+++ b/ip/link_gre.c
@@ -12,11 +12,12 @@
 
 #include <string.h>
 #include <net/if.h>
-#include <linux/if_tunnel.h>
 #include <sys/types.h>
 #include <sys/socket.h>
 #include <arpa/inet.h>
 
+#include <linux/ip.h>
+#include <linux/if_tunnel.h>
 #include "rt_names.h"
 #include "utils.h"
 #include "ip_common.h"
